etc/dosemu.conf

```diff
@@ -765,6 +765,16 @@
 
 # $_SDL_hwrend = (off)
 
+# Name of the TTF font that is used for SDL2 (e.g. "vga")
```

Comment:
google seem to suggest SDL_ttf can render bdf, pcf
fonts as well. Not very important, but have you tried
it with our currently provided fonts?

Reply:
No I didn't try this yet. Will have to figure out the window sizing differently in that case.

Reply:
I tried the example program `showfont` that comes with SDL_ttf. It seems to render pcf.gz okay, but has a problem with vertical alignment on bdf
![bdf](https://user-images.githubusercontent.com/220835/59145994-74b34d00-89e3-11e9-8fbe-35a0230b14d9.png)